pubspec.yaml

```diff
@@ -33,9 +34,13 @@ flutter:
 
   # To add assets to your package, add an assets section, like this:
   assets:
-
-    # - images/
-    - assets/images/
+    - packages/feed_sx/assets/images/avatar.png
```

Comment:
We have to create the assets file out of the lib directory for the files to be globally accessible, and like I had asked you previously too why do we have an extra "src" folder. It seems unnecessary.

Reply:
https://dart.dev/tools/pub/package-layout#implementation-files

Reply:
Okay, but this is not a package we are building. The package will be the SDK. Rest it is your choice completely. what about the assets folder?

Reply:
Let's discuss this point over a call. I have some doubts. Thanks for pointing that out. 
As far as the assets folder is concerned, I implemented that from a Package's perspective.